### ISSUE TITLE
I don't have IntelliSense.

### ISSUE THREAD

**Issue by andrewio** [dotBunny/VSCode#19]
I followed all the instructions(https://code.visualstudio.com/Docs/runtimes/unity), but IntelliSense still does not work

**Comment by LichiMan** [comment]
Same here. I'm on OSX 10.10.5, VS Code 0.8.0 and Unity 5.2.1
By the way, Unity doesn't create the -CSharp version of the solution file anymore, but the VS Code flame is "burning" with the solution file without -CSharp.

**Comment by reapazor** [comment]
I havent been able to track this down; however a little bird mentioned you might not have the latest mono installed?

**Comment by LichiMan** [comment]
Yes, I do. I didn't get any OmniSharp errors, but I installed it like the documentation says:
VS Code uses a more recent version of Mono than that included with Unity. If you get OmniSharp errors, you may need to update your Mono version. See this FAQ topic for additional details.

**Comment by reapazor** [comment]
Any special characters in the path to the project?

**Comment by LichiMan** [comment]
Ok. It works now.
I installed Mono from http://www.mono-project.com/download/ as you say in the Readme file and IntelliSense didn't work.
Now I've installed Mono with brew, as it says in the Visual Studio Code documentation and it's working now.
The weird thing is that it didn't show up any errors. Anyway, I'm going to test it right now. :)

Thanks for doing this!

**Comment by reapazor** [comment]
What i'm finding/seeing often is depending on OS/permissions the path to mono isnt being set in your bash profile. The Homebrew version might manually put mono in the /usr/bin folder, which in the new 10.11 OSX is not allowed (or discouraged). Anyways ... I found that by adding it to my PATH i was able to get it working. I'll probably update some documentation to talk about the 10.11 OSX changes.

**Comment by LichiMan** [comment]
I've the 10.11 OSX installed and Homebrew worked just with:
brew update
brew doctor
brew reinstall mono

**Comment by reapazor** [comment]
In 10.11 there was some changes to what they recommend for executable placement. 

The lastest versions of Mono respect that, and symlink in th /usr/local/bin folder. However at times, you might need your bash_profile updated as well to point to the proper framework (current) folder.

I also found when I updated my /usr/local/bin had lost its permissions and I had to set that manually.

So this looks like a 'complex' problem where everyones solution might be slightly tailored to their own computers (which is gonna suck till it is officially sorted out).


When I looked at Homebrew, they haven't updated to respect the wishes of 10.11 yet, which is a good thing in this case :)
